app.rb

```diff
@@ -1,9 +1,24 @@
 require 'sinatra/base'
+require_relative './lib/peep_messages'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/' do 
```

Comment:
Regarding your routes names here, you could be more explicit. Instead of ‘/messages‘ - to be more consistent with the theme of the app you could have: 

**post /peeps do** 

Also, your **get '/' do** isn’t wrong but if you wanted to be more explicit you could do:

**get '/messages' do** or **get ‘/peeps’ do**

Hopefully that makes sense, if not then please do DM me to ask me questions :) 

More info on restful routes can be found here: https://github.com/makersacademy/course/blob/main/pills/rest.md 

Regarding TDD - I would have liked to have seen a test driven approach as it looks like you have tests missing, do you know have you could have approached this in a more test driven way? I’d like to know via DM please :D 

I can see that you’ve complete the first few user stories which is good. What would your next steps be to continue with this challenge? You can DM me with these if you’d like.